llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -887,8 +902,8 @@ static void computeImportForFunction(
           // If insert doesn't happen, there must be an existing entry keyed by
```

Comment:
nit: could consider shortening the comment. It might be that only the "Note `ExportLists` only keeps track of exports due to imported definitions." is useful now?

Reply:
Done.  Thanks!